src/migrations/20190828145652-create-request.js

```diff
@@ -0,0 +1,60 @@
+/* eslint-disable arrow-parens */
+
+module.exports = {
+  up: (queryInterface, Sequelize) => queryInterface.createTable('Requests', {
+    id: {
+      allowNull: false,
+      autoIncrement: true,
+      primaryKey: true,
+      type: Sequelize.INTEGER
+    },
+    origin: {
```

Comment:
I am confused, most of your team members are using `source` and you have `origin`, please which one is the current field name?

Reply:
I'll change it back to source